packages/react-dev-utils/ModuleScopePlugin.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
```

Comment:
Just curious all project under MIT license, but some pieces of code under BSD-style license?